src/features/wallet/scan/ScanView.tsx

```diff
@@ -6,6 +6,7 @@ import { useTranslation } from 'react-i18next'
 import { useSelector } from 'react-redux'
 import { BarCodeScanner, BarCodeScannerResult } from 'expo-barcode-scanner'
```

Comment:
Can we remove `expo-barcode-scanner` entirely now? I believe we get everything we need from 
`import { Camera, BarCodeScanningResult } from 'expo-camera'`

Reply:
It looks like we still need it for the type to pass to https://docs.expo.dev/versions/latest/sdk/camera/#barcodescannersettings